package.json

```diff
@@ -52,7 +52,9 @@
         "start": "yarn build && cd examples/query-playground && yarn start",
         "docs:build": "d2-utils-docsite build ./docs -o ./dist && yarn build:playground",
         "docs:serve": "d2-utils-docsite serve ./docs -o ./dist",
-        "lint": "d2-style check js && d2-style check text"
+        "lint": "d2-style check js && d2-style check text",
+        "link:all": "yarn workspaces run link",
+        "unlink:all": "yarn workspaces run unlink"
```

Comment:
I set these up to make working with the UI library easier - I think if might be worth keeping them in for future work (thought it means having `lint` scripts in the all services package.json files too)

Means that running `yarn link:all` in app-runtime and then in ui running `yarn link:app-runtime` should get them to work together nicely. Happy to remove them though if there is any objection